components/autofill/src/sync/address/mod.rs

```diff
@@ -4,13 +4,17 @@
 */
 
 pub mod incoming;
+pub mod outgoing;
 
 use super::engine::{ConfigSyncEngine, EngineConfig, SyncEngineStorageImpl};
-use super::{MergeResult, Metadata, ProcessIncomingRecordImpl, SyncRecord};
+use super::{
+    MergeResult, Metadata, ProcessIncomingRecordImpl, ProcessOutgoingRecordImpl, SyncRecord,
+};
 use crate::db::models::address::InternalAddress;
 use crate::error::*;
 use crate::sync_merge_field_check;
-use incoming::AddressesImpl;
+use incoming::AddressesImpl as IncomingAddressesImpl;
```

Comment:
If you prefer, feel free to rename the struct in incoming.rs and outgoing.rs and avoid the `use ... as` here - up to you.